### ISSUE TITLE
Monitor in Portrait Position

### ISSUE THREAD

**Issue by RobertJasiek** [rooklift/ogatak#103]
I use Ogatak 1.9.0_windows in Windows 11 Pro 64b. My monitor has the resolution 1024 x 1280 in portrait position.

Ogatak does not show the window and its contents correctly when the window is maximised or manually enlarged to the monitor boundaries.

Apparently, Ogatak makes the mistake to assume the monitor in landscape position with the resolution 1280 x 1024. Accordingly, the right part of the window and board are truncated in the displayed view. Ogatak needs to check whether the monitor is in portrait or landscape position.

**Comment by rooklift** [comment]
True, it wasn't really designed with portrait orientation in mind... I'll have to think about this.
